### PR TITLE
Fix exotics calculation for loadout optimizers.

### LINKED ISSUE #2135: Max loadout ignores exotic limit
When pressing max loadout, DIM ignores the max exotic limit.  e.g. it tried to equip more than 1 exotic weapon.  I would assume this affects the visible max light level too.

### PR DESCRIPTION
**Pull request by bhollis**
Fixes #2135

### CODE REVIEW

File: src/app/loadout/loadout-popup.component.js [hunk 1]
```diff
@@ -410,7 +410,10 @@ function LoadoutPopupCtrl($rootScope, $scope, ngDialog, dimLoadoutService, dimIt
     });
 
     // Solve for the case where our optimizer decided to equip two exotics
-    const exoticGroups = [['Primary', 'Special', 'Heavy'], ['Helmet', 'Gauntlets', 'Chest', 'Leg']];
+    // TODO: D2 gives us a way better way to do this with equippingBlock info, but it's too complex to figure out now
+    const exoticGroups = vm.store.destinyVersion === 1
```
Comment (kyleshay) [inline src/app/loadout/loadout-popup.component.js]: ahh, that makes sense. nice catch

**Comment by paulharman** [comment]
I think there is still an issue with the calculation of max light level label (shown to the right of Maximise light level). Mine shows it as 287 however when I maximise my light I get to 286

**Comment by bhollis** [comment]
Yeah, I think the different pieces don't weight exactly the same. 

-Ben

**Comment by bhollis** [comment]
Fixed light calculation in #2157.

**Pull request merged**